backend/api_gateway/src/main/kotlin/com/linkedout/backend/service/CompanyService.kt

```diff
@@ -16,7 +15,7 @@ class CompanyService(
     @Value("\${app.services.companies.subjects.findOne}") private val findOneSubject: String
 ) {
     fun findAll(requestId: String): Flux<Company> {
```

Comment:
You can also remove the `Flux` here to return a `List`. Then, transform this into a `Flux` in the controller.
```suggestion
    fun findAll(requestId: String): List<Company> {
```